services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -377,7 +377,7 @@ async def list_projects(  # pylint: disable=too-many-arguments,too-many-statemen
         # attribute filters
         filter_by_project_type: ProjectType | None = None,
         filter_by_services: list[dict] | None = None,
-        filter_published: bool | None = False,
+        filter_published: bool | None = None,
```

Comment:
NOTE: perhaps we should use UNSET instead of None to stress the meaning of the defaults in these cases